aten/src/ATen/core/ivalue.cpp

```diff
@@ -118,4 +118,24 @@ void ivalue::Object::resizeObject(size_t slot) {
   slots_.resize(type()->numAttributes());
 }
 
+static bool CompareIValue(const std::pair<IValue, IValue>& aWrap,
```

Comment:
`CompareIValue` is a very misleading name. If I saw a comparison function which takes a pair, I would assume it takes the second component into account as well. In here, you completely ignore this. Maybe rename this to `CompareKeys`?